src/tiled/editpolygontool.h

```diff
@@ -98,6 +108,7 @@ private slots:
     QPointF mStart;
     QPoint mScreenStart;
     Qt::KeyboardModifiers mModifiers;
+    bool mExtendingFirst;
```

Comment:
I know this variable isn't expected to be used before being set, but please anyway initialize the variable.